### PR TITLE
Added support for including an optional chain cert with the tomcat ssl cert.

### PR DESCRIPTION
**Pull request by zmoore-usgs**
Using steps from: https://medium.com/java-user-group-malta/importing-existing-certificates-into-a-keystore-using-openssl-905d9e0b7175

### CODE REVIEW

File: config.env [hunk 1]
```diff
@@ -5,6 +5,7 @@ keystoreSSLKey=tomcat
 serverPort=8443
 serverContextPath=/
 SPRING_CLOUD_CONFIG_ENABLED=false
+TOMCAT_CHAIN_PATH=/home/spring/tomcat-wildcard-ssl.chain
```
Comment (isuftin) [inline config.env]: Can we be sure that there will only ever be one chain cert?

Comment (zmoore-usgs) [reply config.env]: Probably not, but perhaps we make it a requirement that if there are multiple chain certs they are combined prior to this? From what I've read the order of the certs when combined into a single file does matter so either we'd need a naming convention that expresses the order to combine them in, or we'd need them pre-combined at this point.

Comment (isuftin) [reply config.env]: I'd leave this as is and leave it up to the implementor of the configuration for spawning the container to perform the combination into a single chain file. Leave as is and revisit if needed down the road.

**Comment by isuftin** [comment]
This should be good to go at this point

**Pull request merged**